server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/ElasticSearchMailboxModule.java

```diff
@@ -100,6 +104,16 @@ private ElasticSearchSearcher createMailboxElasticSearchSearcher(Client client,
             MailboxElasticSearchConstants.MESSAGE_TYPE);
     }
 
+    @Provides
+    @Singleton
+    private ElasticSearchHealthCheck createElasticSearchHealthCheck(Client client, ElasticSearchConfiguration configuration) {
+        Set<IndexName> indices = new HashSet<>();
+        indices.add(configuration.getIndexMailboxName());
+        indices.add(configuration.getIndexQuotaRatioName());
+        return new ElasticSearchHealthCheck(
+                client,indices,1000);
```

Comment:
Extract `1000` in a named constant.

Also we tend to put a space after coma. I think it might also break the checkstyle...